### ISSUE TITLE
Change Languages to be Like Ideologies A, AB, B, C

### ISSUE THREAD

**Issue by lkettle** [hashkat/hashkat#80]
Doing web development on Hashkat.  

It would be nice if user could put own names on languages.  

Ideally, it would be nice if could have 3 languages, A, B, C, then combinations AB, BC, AC, ABC which user could put their own add weights on.

**Comment by itamblyn** [comment]
Agreed.

**Comment by kryczko** [comment]
Yes, I believe the data structures implemented currently DO NOT sure bilingualism properly.
